gguf-py/gguf/gguf_reader.py

```diff
@@ -104,7 +106,7 @@ class ReaderTensor(NamedTuple):
     n_elements: int
     n_bytes: int
     data_offset: int
-    data: npt.NDArray[Any]
+    data: np.ndarray
```

Comment:
```suggestion
    data: np.ndarray[Any, Any]
```

Keep the annotations where possible when changing `npt.NDArray` to `np.ndarray`.

Reply:
actually given the type fix I added to this file, I should have reverted this change entirely. Depending on final say from you, I'll either revert it or do as you say here when I roll back the pyright hint at the top.